default.xml

```diff
@@ -26,15 +26,24 @@
   <project path="packages/apps/Launcher3" name="platform/packages/apps/Launcher3" groups="pdk-fs" remote="aosp" />
   <project path="packages/apps/LineageParts" name="android_packages_apps_LineageParts" remote="maruos" />
   <project path="prebuilts/busybox" name="platform_prebuilts_busybox" remote="maruos" revision="master" />
-
+  <project name="LineageOS/android_packages_resources_devicesettings" path="packages/resources/devicesettings" remote="github" revision="lineage-16.0" />
 
   <project path="vendor/maruos" name="vendor_maruos" remote="maruos" />
   <project path="vendor/maruos/mflinger" name="mflinger" remote="maruos" />
   <project path="vendor/maruos/blueprints" name="blueprints" remote="maruos" revision="master" />
   <project path="vendor/maruos/installer" name="installer" remote="maruos" revision="master" />
 
   <!-- Devices -->
+
+  <!-- START hammerhead - Nexus 5 -->
   <project path="kernel/hammerhead" name="android_kernel_hammerhead" groups="hammerhead" remote="maruos" clone-depth="100"/>
   <project path="device/lge/hammerhead" name="android_device_lge_hammerhead" groups="device,hammerhead,broadcom_pdk,generic_fs" remote="maruos" />
+  <!-- END hammerhead - nexus 5 -->
+
+  <!-- START pme - HTC 10 -->
+  <project path="device/htc/pme" name="android_device_htc_pme" remote="pintaf" />
```

Comment:
Maybe you should push those projects to maruos. Hi @pdsouza , could you grant permission to @pintaf ?

Reply:
Yes, I will add @pintaf to our team.